packages/web-runtime/src/pages/resolvePublicLink.vue

```diff
@@ -183,7 +187,7 @@ export default defineComponent({
   },
   methods: {
     async resolvePublicLink(passwordRequired) {
-      if (this.tokenInfo?.alias_link) {
+      if (this.tokenInfo?.alias_link || this.isUserContext) {
```

Comment:
This won't work for oC10 as the condition will evaluate to `true` in an authenticated context, without having `tokenInfo`.